pilot/cmd/pilot-agent/main.go

```diff
@@ -681,7 +710,10 @@ func init() {
 
 	proxyCmd.PersistentFlags().Uint16Var(&statusPort, "statusPort", 0,
 		"HTTP Port on which to serve pilot agent status. If zero, agent status will not be provided.")
-
+	proxyCmd.PersistentFlags().IntVar(&stsPort, "stsPort", 0,
+		"HTTP Port on which to serve pilot agent status. If zero, agent status will not be provided.")
```

Comment:
wrong description

Reply:
Nice catch. I have updated the description. Thanks.